book/src/concepts/advanced_replication/prediction.md

```diff
@@ -42,6 +42,38 @@ Then on the server, you can replicate back the entity to the original client. Th
 instead of spawning a new separate `Predicted` entity,
 it will re-use the existing entity that had the `PrePredicted` component!
 
+## Which components should I predict?
```

Comment:
I think it would be helpful to add a short intro here:

you want to predict components for entities that will live in the `Predicted` timeline, i.e. the timeline that will see changes immediately based on client inputs. However it doesn't mean that every component needs to be actively be predicted with `ComponentSyncMode::Full`, i.e. with rollbacks enabled. A lot of components can be computed from other components and don't need to be predicted or even sent through the network. Here is a quick explanation of which components should be predicted